cdap-app-templates/cdap-etl/hydrator-spark-core/src/main/java/io/cdap/cdap/etl/spark/batch/RDDCollection.java

```diff
@@ -72,8 +80,8 @@ public RDDCollection(JavaSparkExecutionContext sec, JavaSparkContext jsc, SQLCon
     collections.put(joinRequest.getLeftStage(), left);
 
     List<Column> leftJoinColumns = joinRequest.getLeftKey().stream()
-      .map(left::col)
-      .collect(Collectors.toList());
+                                              .map(left::col)
```

Comment:
style: IDE settings seem to be off, method calls should be indented, not aligned to the previous line

Reply:
Yes..I've fixed them now but I have a feeling that some of our other repos have them aligned not indented. I find myself repeatedly toggling this setting

Reply:
method calls should be indented, but if you're in a middle of a method call, the arguments should be aligned.

```
  x.stream()
    .map()
```

vs 

```
  x.call(a,
         b);
```

Reply:
Ah gotcha, that might be where the confusion came from